crates/client/README.md

```diff
@@ -64,23 +64,23 @@ assert!(!a_data.is_empty());
 
 ## DNS-over-TLS and DNS-over-HTTPS
 
-DoT and DoH are supported. This is accomplished through the use of one of `native-tls`, `openssl`, or `rustls` (only `rustls` is currently supported for DoH).
+DoT and DoH are supported. This is accomplished through the use of `rustls`.
 
 To use DoT or DoH with the `Client`, construct it with `TlsClientStream` or
 `HttpsClientStream`. Client authentication/mTLS is currently not supported,
 there are some issues still being worked on. TLS is useful for Server
 authentication and connection privacy.
 
-To enable DoT, one of the features `dns-over-native-tls`, `dns-over-openssl`, or
-`dns-over-rustls` must be enabled. `dns-over-https-rustls` is used for DoH.
+To enable DoT, one of the features `dns-over-rustls` must be enabled. `dns-over-https-rustls` is used for DoH. `dns-over-h3` is used for dns over http3 and `dns-over-quic`
```

Comment:
And instead, turn this into a bullet list, like this:

```markdown
Some protocols are optionally supported:

- Enable `dns-over-rustls` for DNS over TLS (DoT)
- Enable `dns-over-https-rustls` for DNS over HTTP/2 (DoH)
- Enable `dns-over-quic` for DNS over QUIC (DoQ)
- Enable `dns-over-h3` for DNS over HTTP/3 (DoH3)
```

Text like this can be copied across the various files.

Reply:
Updated, should I update the other REAMD in a similar way as well?

Reply:
Yes, please!

Reply:
Thanks updated